### DISCUSSION TITLE
How to create subtasks for a task from jira-cli

### DISCUSSION THREAD

**Discussion by MRColorR** [ankitpokhrel/jira-cli#744] [Q&A] [answered]
Hello everyone,
I'm looking to understand how to create a subtask for an existing task. I've been able to manage tasks through the CLI but haven't figured out the command for subtasks yet. i tried with something like `jira issue create -p <project> -P <parent-task>  -t subtask -s"<summary>"` but i got "error 400 - issue type is required"

Does anyone have experience with this or know the correct command syntax to achieve this? Any pointers or examples would be greatly appreciated.
Tysm

**Comment by ankitpokhrel** [comment]
is `subtask` the valid issue type? You need to pass a valid type

**Reply by ryanpeach** [reply]
Is it? What is the valid type for this?

**Reply by ryanpeach** [reply, marked as answer]
For the record this is the proper type, took forever to find. Case and string matters `-t'Sub-task'`
